doc/nrf/includes/mesh_ext_flash_settings.txt

```diff
@@ -0,0 +1,18 @@
+External flash for settings partition
+#####################################
+
+This sample supports relocating mesh settings to the MX25R64 QSPI NOR external flash on ``nrf52840dk/nrf52840``.
```

Comment:
I do not think we need to mention memory type and interface that it uses in mesh documentation. We need to put reference to DK documentation instead. We, as a team, do not follow up DK changes. This information might be deprecated without letting us know.